src/videojs-contrib-hls.js

```diff
@@ -278,8 +279,42 @@ export default class HlsHandler extends Component {
       let seekable;
 
       if (!updatedPlaylist) {
+        let selectedPlaylist = this.selectPlaylist();
+
+        if (!selectedPlaylist.attributes.AUDIO) {
+          // from safari defaults
+          this.tech_.audioTracks().addTrack(new AudioTrack({
+            enabled: true,
+            id: '1',
+            kind: 'main',
+            tech: this.tech_
+          }));
+        } else {
```

Comment:
No action necessary but when the branches of conditionals get this big I tend to favor short-circuit `return`s so you don't have to remember what block you're in for so long while reading.